tests/phpunit/CRM/Member/Form/MembershipTest.php

```diff
@@ -545,6 +546,14 @@ public function testSubmit($thousandSeparator) {
       'Receipt text',
     ));
     $this->mut->stop();
+    $this->assertEquals([
+      [
+        'text' => 'AnnualFixed membership for Mr. Anthony Anderson II has been added. The new membership End Date is December 31st, ' . date('Y') . '. A membership confirmation and receipt has been sent to anthony_anderson@civicrm.org.',
```

Comment:
Is this one of those tests that will fail once a year on Dec 31?

Reply:
I can cope with the once a year ones - it's the once a month ones that bother me

Reply:
What's the expected behavior? That an annual membership beginning Dec 31 expires this year or next? Why not set the start date and end date in the test and avoid these quirks?

Reply:
OK - I've hard-coded the month to Feb which should keep it reliable I think - never 31 Dec